sdk/js/src/token_bridge/getForeignAsset.ts

```diff
@@ -191,22 +191,37 @@ export async function getForeignAssetNear(
   return ret !== "" ? ret : null;
 }
 
+/**
+ * Get qualified type of asset on Aptos given its origin info.
+ * @param client Client used to transfer data to/from Aptos node
+ * @param tokenBridgeAddress Address of token bridge
+ * @param originChain Chain ID of chain asset is originally from
+ * @param originAddress Asset address on origin chain
+ * @returns Fully qualified type of asset on Aptos
+ */
 export async function getForeignAssetAptos(
   client: AptosClient,
   tokenBridgeAddress: string,
   originChain: ChainId | ChainName,
   originAddress: string
 ): Promise<string | null> {
   const originChainId = coalesceChainId(originChain);
-  const assetAddress = getForeignAssetAddress(tokenBridgeAddress, originChainId, originAddress);
-  if (!assetAddress) {
+  const assetFullyQualifiedType = getAssetFullyQualifiedType(
+    tokenBridgeAddress,
+    originChainId,
+    originAddress
+  );
+  if (!assetFullyQualifiedType) {
     return null;
   }
 
   try {
     // check if asset exists and throw if it doesn't
-    await client.getAccountResource(assetAddress, `0x1::coin::CoinInfo<${ensureHexPrefix(assetAddress)}::coin::T>`);
-    return assetAddress;
+    await client.getAccountResource(
+      getModuleAddressFromFullyQualifiedType(assetFullyQualifiedType),
+      `0x1::coin::CoinInfo<${assetFullyQualifiedType}>`
+    );
+    return assetFullyQualifiedType;
```

Comment:
This is a subtle but breaking change for integrators, no? Is there a reason this change is recommended over what was working?

Reply:
Discussed and this feels right, so we'll minor version bump. @heyitaki can you update the CHANGELOG and package.json to reflect this?

Reply:
done